### PR TITLE
Add script to downstream PRs.

### PR DESCRIPTION
**Pull request by twiecki**

### CODE REVIEW

**Review by ricardoV94 — APPROVED** [review]

**Comment by ferrine** [comment]
I think a more descriptive or message is needed, very hard to figure out what the or was about

**Review by ferrine — CHANGES_REQUESTED** [review]
Needs a more descriptive pr title

**Review by ferrine — COMMENTED** [review]
I meant this informative title
```bash
curl https://api.github.com/repos/aesara-devs/aesara/pulls/1271 2>/dev/null | jq '.title'
"Implement multi-output Elemwise in Numba via guvectorize"
```

File: bin/downstream_pr.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+else
+    echo "Patch failed. Find the .rej file and apply the changes manually. Then 'git add' all changed files, followed by 'git am --continue'. Then create a PR manually."
+    exit 1
+fi
+
+echo "Running pre-commit"
+pre-commit run --all
+
+git push origin downstream_$1
+
```
Comment (ferrine) [inline bin/downstream_pr.sh:41]:
```suggestion
title=$(curl https://api.github.com/repos/aesara-devs/aesara/pulls/1271 2>/dev/null | jq '.title')
```

File: bin/downstream_pr.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+gh pr create --repo pymc-devs/pytensor --title "Downstreaming Aesara PR $1" --body "Downstreaming https://github.com/aesara-devs/aesara/pull/$1. PR port done by downstream_pr.sh script."
```
Comment (ferrine) [inline bin/downstream_pr.sh:42]:
```suggestion
gh pr create --repo pymc-devs/pytensor --title "Downstreaming Aesara PR $1: $title" --body "Downstreaming https://github.com/aesara-devs/aesara/pull/$1. PR port done by downstream_pr.sh script."
```

**Review by ferrine — APPROVED** [review]

**Pull request merged**